src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
 
 const ChatEntry = (props) => {
+  const { id, sender, body, timeStamp, liked, onToggleLike, names, textColor } = props
+  const heartDisplay = liked ? '❤️' : '🤍'
+  const textClass = sender === (names && names[0]) ? 'local' : 'remote';
+  const textColorClass =
+    sender === (names && names[0]) ? textColor[names[0]] : (names && textColor[names[1]]);
```

Comment:
By doing this work here in ChatEntry, the component needs to know about multiple senders and handling how to present info for a particular sender. I would consider moving this logic up to the ChatLog (which already needs to take in messages for multiple senders) and passing the textClass and textColorClass directly to ChatEntry.